docs/set_up_admin_tails.rst

```diff
@@ -66,6 +66,11 @@ First, download and verify the **SecureDrop Release Signing Key**.
           copy-pasting this command, we recommend you double-check you have
           entered it correctly before pressing enter.
 
+.. note:: If the command hangs and evenutally fails with a timeout, it
+          could indicate the GPG key server is down or unreachable. As
+          a workaround, another keyserver can be specified: for
+          instance ``--keyserver hkp://pool.sks-keyservers.net``.
```

Comment:
I would add a little more detail to this sentence so to try to make it clearer to SecureDrop administrators who may be unfamiliar with GPG, e.g: "As a workaround, another keyserver can be specified by adding the `--keyserver` option to the `gpg --recv-key` command. In our experience, the SKS keyserver pool is usually a reliable alternative, so try re-running the command with `--keyserver hkp://pool.sks-keyservers.net`".